runnumber-rest/entrypoint.sh

```diff
@@ -1,15 +1,8 @@
 #!/bin/bash
 
-
 cd $(dirname $0)
 source ../entrypoint_functions.sh
 
-ensure_required_variables "RNURI RNUSER RNPASS"
-
-export LD_LIBRARY_PATH=/usr/lib/oracle/12.1/client64/lib/:$LD_LIBRARY_PATH
```

Comment:
If retaining Oracle compatibility is required, this `LD_LIBRARY_PATH` bit will probably still be needed in addition to whatever package provides the client libraries.